apps/perf-test/src/index.scenarios.tsx

```diff
@@ -40,11 +41,11 @@ if (renderType === 'virtual-rerender') {
   // It'd be interesting to root cause why at some point.
   // ReactDOM.render(<>{Array.from({ length: iterations }, () => (scenarios[scenario]))}</>, div);
   ReactDOM.render(
-    <div>
+    <PerfTestDecorator>
       {Array.from({ length: iterations }, () => (
         <PerfTestScenario />
       ))}
-    </div>,
+    </PerfTestDecorator>,
```

Comment:
@ecraig12345 is it okay or do you have better ideas?

Reply:
Better ideas about which part?

Reply:
_Oops, IDK where I lost the first part of message 🦆_ 

---

Currently components that are consuming `makeStyles()` require to have a wrapping `Provider` component always, otherwise they will throw an exception. _We can and should reconsider this._

However, currently I would like to proceed with this PR. I don't think that it's correct to wrap all perf examples with converged `Provider`, so I decided to implement decorators that can be specified in examples. In this case, only examples that have `.decorator` will be wrapped. This approach also leaves us some flexibility for future.

So, is it okay or do you have better ideas? 😄

Reply:
I'm actually not that familiar with the perf tests, so I don't have any better ideas offhand. This generally makes sense to me though.